### DISCUSSION TITLE
Is rooftop solar included in solar power production?

### DISCUSSION THREAD

**Discussion by TheMcSquirrell** [electricitymaps/electricitymaps-contrib#4940] [Q&A] [answered]
I hope my question is clear. Rooftop solar takes place at the distribution level, and in some countries makes up a significant share of generation. Does it contribute to a reduced consumption, or is it included in energy generation?

**Comment by VIKTORVAV99** [comment, marked as answer]
There is no one answer to this, we get our data from TSOs and national grid operators. Some of these TSOs and grid operators include estimated rooftop solar in their figures, some don't and sadly there is not much we can do about that.

Hopefully this makes sense and answers you question.
